docs/notebooks/diagnostics/line_integrated_diagnostic.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>As a newbie to interferometry, and speaking for all newbies to interferometry to come after me - some more explanation of the unwrapping could be handy! :D I have a hunch about how it works, and I expect diving through the code will confirm it, but just noting a note of confusion down.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/1123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
So, basically you just remove discontinuities of 2pi using `np.unwrap`. 

I think this belongs in the Interferometer notebook, so I'm starting that now and I'll manually unwrap the signal there as an example.